test/quantization/test_quantized.py

```diff
@@ -369,30 +396,7 @@ def test_hardtanh(self, X, min_val, max_val):
                        elements=hu.floats(-1e6, 1e6, allow_nan=False, allow_infinity=False),
                        qparams=hu.qparams()))
     def test_hardswish(self, X):
-        X, (scale, zero_point, torch_type) = X
-        X = torch.from_numpy(X)
-        qX = torch.quantize_per_tensor(X, scale=scale, zero_point=zero_point,
-                                       dtype=torch_type)
-        dqX = qX.dequantize()
-
-        output_scale = scale
-        output_zero_point = zero_point
-
-        dqY_hat = F.hardswish(dqX)
-        qY_hat = torch.quantize_per_tensor(dqY_hat, scale=output_scale,
-                                           zero_point=output_zero_point,
-                                           dtype=torch_type)
-
-        # regular
-        qY = torch.nn.quantized.functional.hardswish(qX)
-        self.assertEqual(qY, qY_hat,
-                         message="Hardswish failed: {} vs {}".format(qY, qY_hat))
-
-        # inplace
-        qX_copy = qX.clone().detach()
-        torch.nn.quantized.functional.hardswish(qX_copy, inplace=True)
-        self.assertEqual(qX_copy, qY_hat,
-                         message="inplace Hardswish failed: {} vs {}".format(qY, qY_hat))
+        _test_hardswish(self, X, 'fbgemm')
```

Comment:
Is there a reason why `test_x` is calling `_test_x`? Maybe rename this test as `test_x_fbgemm`? Or use a hard-coded parameter?

Reply:
it was a way to share the test code from the two different test classes, default and qnnpack (with different behaviors on what to skip per platform).  The test module is a part of the name during runs.  I'm flexible on if we can make it even better somehow - lmk!